sdk/Cargo.toml

```diff
@@ -144,7 +144,7 @@ unic-langid = { version = "0.9.1", features = ["serde"], optional = true }
 rustc-hash = { version = "1.1.0", optional = true }
 postcard = { version = "1.0.2", features = ["use-std"], optional = true }
 once_cell = { version = "1.17.0", optional = true }
-dioxus-signals = { version = "0.5.0-alpha.2", features = [
+dioxus-signals = { git = "https://github.com/DioxusLabs/dioxus", branch = "v0.5", features = [
```

Comment:
My only concern is if we'd want to do an early release of SDK. I don't think we will but this would postpone SDK until Dioxus 0.6. Probably merge anyway if there aren't any other issues.

@marc2332 Thoughts?

Reply:
I thought I reverted that change. The storage example needs to patch the dioxus version for fullstack, but it doesn't require any breaking changes, so we don't need to change the main sdk version